### PR TITLE
update font config.ini to include OS7 stuff

### PR DESCRIPTION
**Pull request by NoSloppy**
Wanted to PR this for review, not just commit.

### CODE REVIEW

File: sound/sound-font-config.ini-configuration.md [hunk 1]
```diff
@@ -25,111 +27,206 @@ volHum=15
...
 
 # This is used to control the volume of the combined hum and smoothswings
-# when an accent swing plays.                       
+# when an accent swing plays.
+# Values range from 0.0 to 1.0, with 0 being no reduction, and 1
+# meaning that when the accent swing is at peak volume, the
+# smooth swing pairs are totally muted.                       
 # Defaults to 0.2 (volume is decreased by 20% of swing volume)    
-ProffieOSSmoothSwingDucking=0.2 
+ProffieOSSmoothSwingDucking=0.2
```
Comment (profezzorn) [inline sound/sound-font-config.ini-configuration.md:56]: extra space at end

File: sound/sound-font-config.ini-configuration.md [hunk 2]
```diff
@@ -25,111 +27,206 @@ volHum=15
...
+# Number of degrees the blade must travel while staying above the
+# swing threshold in order to trigger a spin sound.
+# Default is 360 or one full rotation.
+ProffieOSSpinDegrees=360.0
+
+# ---- As of ProffieOS 6 ---- #
+
+# Minimum acceleration for Accent Swing file Selection
+# recommended value is 20.0 ~ 30.0
+ProffieOSMinSwingAcceleration=20.0
```
Comment (profezzorn) [inline sound/sound-font-config.ini-configuration.md:80]: Missing a "default" line.

File: sound/sound-font-config.ini-configuration.md [hunk 3]
```diff
@@ -25,111 +27,206 @@ volHum=15
...
+
+# Maximum acceleration for Accent Swing file Selection
+# must be higher than Min value to enable selection
+# recommended value is 100.0 ~ 150.0
```
Comment (profezzorn) [inline sound/sound-font-config.ini-configuration.md:84]: Missing default line.

File: sound/sound-font-config.ini-configuration.md [hunk 4]
```diff
@@ -25,111 +27,206 @@ volHum=15
...
+ProffieOSMaxSwingAcceleration=100.0
+
+# Set the volume for each effect individually, in percent.
+# 50 makes it half as loud. 200 makes it twice as loud. 
+# Maximum allowed value is currently 255. The default is 100.
+# EFFECTNAME can be "clash", “preon”, “out”, “pstoff”, etc.
+#
+# ProffieOS.SFX.EFFECTNAME.volume=100
+ProffieOS.SFX.boot.volume=75
```
Comment (profezzorn) [inline sound/sound-font-config.ini-configuration.md:93]: In this file, every variable should be set to it's default value.

File: sound/sound-font-config.ini-configuration.md [hunk 5]
```diff
@@ -25,111 +27,206 @@ volHum=15
...
+
+# Make smoothswings start in sync with hum.
+# Set to 1 to sync, or 0 to resume swings where last pair left off.
+ProffieOSSmoothSwingHumstart=1
```
Comment (profezzorn) [inline sound/sound-font-config.ini-configuration.md:97]: Missing default line.

File: sound/sound-font-config.ini-configuration.md [hunk 6]
```diff
@@ -25,111 +27,206 @@ volHum=15
...
+#-----------------
+# Below are the display durations.
+# If commented out or not specified, the default is used.
+# As of ProffieOS 7, effect IMAGES can use 0.0 to automatically 
+# sync to the effect sound file duration.
 
-# For OLED displays, the time an on.bmp will play
-ProffieOSOnImageDuration=5000.0
+# Text Message duration
+# Default -1.0
```
Comment (profezzorn) [inline sound/sound-font-config.ini-configuration.md:168]: When the default is -1, we need to explain what that actually means.

File: sound/sound-font-config.ini-configuration.md [hunk 7]
```diff
@@ -25,111 +27,206 @@ volHum=15
...
+# same hum sound until next ignition instead of randomly 
+# choosing a different file once the current hum ends,
+#ProffieOS.SFX.hum.paired=1
+
+# Set smoothswings to start in sync with hum start.
+# Set to 1 to sync, or 0 to resume swings where last pair left off.
+# Default 0
+ProffieOSSmoothSwingHumstart=1
+
+# ---- OLED displays ---- #
```
Comment (profezzorn) [inline sound/sound-font-config.ini-configuration.md:141]: So, are all the variables below here OS7 only?
Are we still under the "As of OS7" heading, or do we start over here? It's not very clear.

File: sound/sound-font-config.ini-configuration.md (hunk 6 above)
Comment (NoSloppy) [reply sound/sound-font-config.ini-configuration.md:168]: Which is.....allowing it to be set to zero?

File: sound/sound-font-config.ini-configuration.md (hunk 1 above)
Comment (NoSloppy) [reply sound/sound-font-config.ini-configuration.md]: fixed

File: sound/sound-font-config.ini-configuration.md (hunk 2 above)
Comment (NoSloppy) [reply sound/sound-font-config.ini-configuration.md]: fixed

File: sound/sound-font-config.ini-configuration.md (hunk 3 above)
Comment (NoSloppy) [reply sound/sound-font-config.ini-configuration.md:87]: fixed

File: sound/sound-font-config.ini-configuration.md (hunk 4 above)
Comment (NoSloppy) [reply sound/sound-font-config.ini-configuration.md]: fixed

File: sound/sound-font-config.ini-configuration.md (hunk 5 above)
Comment (NoSloppy) [reply sound/sound-font-config.ini-configuration.md]: fixed

File: sound/sound-font-config.ini-configuration.md (hunk 7 above)
Comment (NoSloppy) [reply sound/sound-font-config.ini-configuration.md]: fixed. How about now?

**Pull request closed without merging**